priv/json-schemas/query-api-schema.json

```
@@ -335,10 +340,29 @@
         }
       ]
     },
+    "filter_for_segment": {
+      "type": "array",
+      "additionalItems": false,
+      "minItems": 3,
+      "maxItems": 3,
+      "items": [
+        {
+          "$ref": "#/definitions/filter_operation_for_segments"
```

Nit: Let's inline the operation given there's only one and we're not repeating the ref anywhere.

```suggestion
          "const": "is"
```